lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py

```diff
@@ -204,7 +204,7 @@ def list_items_by_resource_group(self):
         try:
             response = self.compute_client.disks.list_by_resource_group(resource_group_name=self.resource_group)
         except CloudError as exc:
-            self.fail('Failed to list items by resource group - {}'.format(str(exc)))
+            self.fail('Failed to list items by resource group - {0}'.format(str(exc)))
```

Comment:
This issue has been fixed in devel and does not need to be included in this PR.

Reply:
rebased, thanks